exercises/csharp/Makefile

```diff
@@ -0,0 +1,7 @@
+bootstrap:
+	sudo curl -L -o dotnet-install.sh https://dot.net/v1/dotnet-install.sh
+	sudo chmod +x dotnet-install.sh
+	./dotnet-install.sh
```

Comment:
Not a blocker, but the `dotnet-install` script is meant for CI/transient installation and not for a development environment - can we install .NET using the system package manager?

Also, since we're specifically targeting .NET 6.0 in the `.csproj` files below, it'd be best to install that version specifically (whether using `dotnet-install` or otherwise).

Reply:
I tried to install `dotnet` through `yum` but it doesn't work on Amazon Linux 2 because the repo is meant for CentOS (which has different package names), and Microsoft does not support `dotnet` for AL2 according to [this issue](https://github.com/dotnet/core/issues/930). There is a workaround provided there but I don't think it's worth wrangling for this purpose. The install is just for the workshop; it shouldn't last for more than a few hours. 

Targeting .NET 6.0 explicitly is a good call.